packages/web/package.json

```diff
@@ -51,6 +51,10 @@
     "verify": "concurrently \"npm:typecheck\" \"npm:lint:fix\""
   },
   "dependencies": {
+    "@tanstack/react-query": "^4.18.0",
```

Comment:
Could you add `"@audius/trpc-server": "*"` here, that will let turborepo know that the web client has a dep on trpc-server!